__tests__/app.test.js

```diff
@@ -315,3 +315,41 @@ describe('/api/articles/:article_id/comments', () => {
         });
     });
 });
+
+describe('/api/comments/:comments_id', () => {
+    describe('DELETE', () => {
+        test('DELETE:204 should delete a comment using given parameterised comment_id', () => {
+            const removedCommentId = 1;
+
+            return request(app)
+            .delete(`/api/comments/${removedCommentId}`)
+            .expect(204)
+            .then(({ body }) => {
+                expect(body).toEqual({});
+                return db.query(`SELECT * FROM comments;`)
+            })
+            .then(({ rows, rowCount }) => {
+                expect(rowCount).toBe(17);
+                rows.forEach((row) => {
+                    expect(row.comment_id).not.toBe(removedCommentId);
```

Comment:
I think this is the very first time I've seen someone test this like this. Well done!!